commitlintplugins.test.ts

```diff
@@ -5,6 +5,13 @@ function runCommitLintOnMsg(inputMsg: string) {
 }
 
 
+test('allow-PascalCase-after-colon-in-title', () => {
```

Comment:
@realmarv this rule is about the rule 'subject-lowercase', so let's rename this test and move it next to where the other subject-lowercase tests are

Reply:
Done!